torch/nn/parameter.py

```diff
@@ -28,3 +28,10 @@ def __repr__(self):
 
     def __reduce_ex__(self, proto):
         return Parameter, (super(Parameter, self), self.requires_grad)
+
+    def to(self, *args, **kwargs):
+        device, dtype, non_blocking = torch._C._nn._parse_to(*args, **kwargs)
+        dtype = dtype if self.data.is_floating_point() else None
+        self.data = self.data.to(device, dtype, non_blocking)
```

Comment:
Do we want to make `.to` modify the `Parameter` in-place? That's not the standard behavior of `Tensor`, so I'd saw it can be a bit unintuitive.

Reply:
`Module` does it in-place, and since `Parameter` is more attributed with the usage of a `Module`, I thought it should be that way.

Reply:
It should be like tensor because it is a subclass of tensor.

Reply:
Thanks for your comments, I have modified the behavior.